build.gradle

```diff
@@ -54,6 +54,8 @@ ext {
     /** Environment Building properties **/
     authEnvDir = project.hasProperty("authoring.root") ? project.property("authoring.root") : "./crafter-authoring/"
     deliveryEnvDir = project.hasProperty("delivery.root") ? project.property("delivery.root") : "./crafter-delivery/"
+    authProcessedResourcesDir = "./tmp/crafter-authoring"
```

Comment:
Is this relative path the right way to go? Should we not use a CRAFTER_HOME env /tmp as a the folder path?

Reply:
`.` is always going to be the `craftercms` project folder, Gradle makes sure to set it up when running the build. `CRAFTER_HOME` doesn't exist until it's set in the sh scripts.